packages/babel-plugin-transform-class-properties/src/index.js

```diff
@@ -145,6 +145,16 @@ export default function ({ types: t }) {
         }
 
         path.insertAfter(nodes);
+      },
+      ArrowFunctionExpression(path) {
+        let classExp = path.get("body");
```

Comment:
Looks good?

Maybe it would be better to use the same logic in `Class` above?

https://github.com/jhen0409/babel/blob/5d0da104d9a5bb58deb176796247b3c807b66e0d/packages/babel-plugin-transform-class-properties/src/index.js#L30-L40

We could just copy it or move it out into a helper function?